### ISSUE TITLE
Crash with Bullet DEBUG  enabled

### ISSUE THREAD

**Issue by capnm** [Zylann/godot_heightmap_plugin#191]
**To Reproduce**
* patch the current Godot's 3.2 tip 
[0004-Enable-Bullet-DEBUG-on-debug-builds.zip](https://github.com/Zylann/godot_heightmap_plugin/files/5090601/0004-Enable-Bullet-DEBUG-on-debug-builds.zip)

* add the HTerain node to the scene.

**Screenshots**
```
ERROR: Map width must be at least 2.
   at: set_data (modules/bullet/shape_bullet.cpp:471)
godot-32x: thirdparty/bullet/BulletCollision/CollisionShapes/btHeightfieldTerrainShape.cpp:59: void btHeightfieldTerrainShape::initialize(int, int, const void *, btScalar, btScalar, btScalar, 
int, PHY_ScalarType, bool): Assertion `heightStickLength > 1' failed.
Abgebrochen

```

**Environment**
linux

**Comment by capnm** [comment]
The Error is also printed in the official 3.2.3.rc3 version:

```
E 0:00:00.676   set_data: Map width must be at least 2.
  <C++-Fehler>  Condition "l_width < 2" is true.
  <C++-Quellcode>modules/bullet/shape_bullet.cpp:471 @ set_data()
  <Stacktrace>  hterrain_collider.gd:26 @ _init()
                hterrain.gd:183 @ _init()
```

**Comment by Zylann** [comment]
Pushed a fix in a24deabbbfd8ead91a03aa2bf7326131c65c2cfb
